src/index.html

```diff
@@ -11,7 +11,176 @@
+          <label for="aad">Full date of birth:</label>
+          <input 
+          type="date" 
+          name="attribute" 
+          required
+          >
+        </div>
+
+        <div class="lastInput">
+          <label for="terms-condition">I accept the term of the agreement</label>
+          <input 
+          type="checkbox" 
+          name="attribute" 
+          id="terms-condition" 
+          required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field block">
+        <legend>Registration</legend>
+        <div class="input">
+          <label for="aae">E-mail:</label>
+          <input 
+          type="email" 
+          name="attribute" 
+          placeholder="email@example.com" 
+          required
+          >
+        </div>
+        <div class="lastInput">
+          <label for="aaf">Password:</label>
+          <input 
+          type="password" 
+          name="attribute" 
+          autocomplete="off" 
+          required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field block">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label for="aag">Do you love cats?</label>
+          <label for="yes">
+            <input 
+            type="radio" 
+            id="yes" 
+            name="radio" 
+            checked
+            >
+            Yes
+          </label>
+          <label for="no">
+            <input 
+            type="radio" 
+            id="no" 
+            name="radio"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="input">
+          <label for="aah">What is your favorite color?</label>
+          <input 
+          type="color" 
+          name="attribute" 
+          required
+          >
+        </div>
+
+        <div class="input">
+          <label for="aba">What time do you go to bed?</label>
+          <input 
+          type="time" 
+          name="attribute" 
+          required
+          >
+        </div>
+
+        <div class="input">
+          <label for="abb">What are your favorite brand of cars?</label>
+          <select name="car" 
+          multiple 
+          required>
+            <option value="Audi">Audi</option>
+            <option value="BMW">BMW</option>
+            <option value="carExpert">Lada</option>
+            <option value="none" disabled></option>
+          </select>
+        </div>
+
+        <div class="lastInput">
+          <label for="abc">How do you rate our work?</label>
+          <input 
+          type="range" 
+          required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form-field block">
+        <legend>Additional info:</legend>
+        <div class="input">
+          <label for="abd">Comments:</label>
+          <textarea 
+          name="comments" 
+          maxlength="300" 
+          autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name="comments" 
            maxlength="300" 
            autocomplete="off"
        ></textarea>
```
Fix it to fix this problem
![image](https://user-images.githubusercontent.com/96409563/222083395-7526a774-5a7d-4d74-9cd9-5de820f7c8f3.png)
![image](https://user-images.githubusercontent.com/96409563/222083435-575eefac-6c3e-4e77-be69-e5292327a502.png)